modular_chomp/code/modules/mob/living/simple_mob/subtypes/vore/plants/pitcher.dm

```diff
@@ -39,17 +39,19 @@ GLOBAL_LIST_INIT(pitcher_plant_lure_messages, list(
 	min_n2 = 0
 	max_n2 = 0
 	minbodytemp = 0
+	meat_type = /obj/item/reagent_containers/food/snacks/pitcher_fruit // Allows pitcher plants to be chopped up and replanted. Probably.
+	meat_amount = 1 // And allows you to replant them should you so please.
 
 	melee_damage_upper = 0 //This shouldn't attack people but if it does (admemes) no damage can be dealt.
 	melee_damage_lower = 0
 
 	armor = list(
-				"melee" = 0,
-				"bullet" = 0,
-				"laser" = -50,
+				"melee" = 30, // STOP TRYING TO HACK IT UP WITH AN AXE TO RESCUE A PERSON INSIDE LIKE OH MY GOD YOU ARE ONE UNLUCKY LAG SPIKE FROM CHOPPING THEIR FUCKING CHEST OPEN
```

Comment:
Now I feel like this is _too little_ but whatever I'll just bump it up if people keep defaulting to killing it with an axe.